### PR TITLE
express-get-json - Jeremy Nguyen

### PR DESCRIPTION
**Pull request by jeremyhnguyen**
Screenshot 2023-10-20 at 10 54 53 AM
Screenshot 2023-10-20 at 10 49 59 AM

### CODE REVIEW

File: express-get-json/server.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    id: 273,
+    name: 'Håkon Wium Lie',
+    course: 'CSS',
+    score: 92,
+  },
+};
+
+app.get('/api/grades', (req, res) => {
+  console.log('GET req running'); // remove later
+  res.json(grades);
```
Comment (RobertGardner) [inline express-get-json/server.ts]: The instructions tell you to respond with an array, not an object.

**Review by RobertGardner — CHANGES_REQUESTED** [review]
Fix the response value.

File: express-get-json/server.ts (hunk 1 above)
Comment (jeremyhnguyen) [reply express-get-json/server.ts]: Screenshot 2023-10-20 at 11 52 15 AM

File: express-get-json/server.ts [hunk 2]
```diff
@@ -31,9 +31,14 @@ const grades: Record<number, Grade> = {
   },
 };
 
+const gradesArray: Grade[] = [];
```
Comment (RobertGardner) [inline express-get-json/server.ts]: These should all go inside the app.get() handler function.

**Review by RobertGardner — APPROVED** [review]
Your express handler functions should all be self-contained. You don't want any global calculations to happen.

Go ahead and merge once you make that change. I'll Approve, though I do think you should make this change.

**Pull request merged**